src/test/regress/expected/multi_mx_alter_distributed_table.out

```diff
@@ -174,8 +174,8 @@ AS $$
 DECLARE
     res INT := 0;
 BEGIN
-    INSERT INTO test_proc_colocation_0 VALUES (dist_key);
-    SELECT count(*) INTO res FROM test_proc_colocation_0;
+    INSERT INTO mx_alter_distributed_table.test_proc_colocation_0 VALUES (dist_key);
```

Comment:
can you explain why we need this change?

Reply:
Without it this change this would now result in errors like:
```diff
 CALL proc_0(1.0);
 DEBUG:  pushing down the procedure
-NOTICE:  Res: 3
-DETAIL:  from localhost:xxxxx
+ERROR:  relation "test_proc_colocation_0" does not exist
+CONTEXT:  PL/pgSQL function mx_alter_distributed_table.proc_0(double precision) line 5 at SQL statement
+while executing command on localhost:57637
 RESET client_min_messages;
```

This test was accidentally depending on the search_path being available in the cached connection (because DDL had happened before). 

I think it's very unlikely that customers depend on this behaviour though. Since it would require first doing DDL before calling a procedure.